packages/catalog-server/src/test/lib/catalog_test.ts

```diff
@@ -84,6 +91,37 @@ test('A second import does nothing', async () => {
   assert.equal(importResult.problems, undefined);
 });
 
+test('Full text search', async () => {
```

Comment:
(I thought I sent this comment with the review, but must have lost it somehow).

Feels like we could do with some more test coverage here. Maybe a test rig/utility function that would make it easier to concisely write the scenarios? Some cases it feels like we should have:

- Search by tag name term
- Search by package term
- Search by entire package name
- Search by entire tag name
- Search by description term
- Tests for the various stemmings/other normalization we expect to be happening -- case sensitivity, removing punctuation, plural normalization, etc.
- A search with both a matching and non-matching term (to confirm it's OR not AND matching)
- Checking that the limit is respected
- No matching results
- No elements in the index at all

Reply:
Agree we should definitely have more coverage. I added a few more tests for now.